extensions/wikia/Lightbox/Lightbox.i18n.php

```diff
@@ -73,11 +73,7 @@
 	'lightbox-share-email-label' => 'Email this image to a friend',
 	'lightbox-share-email-page-label' => 'Email this page to a friend',
 	'lightbox-share-email-page-label-address' => 'Email address',
-	'lightbox-share-email-ok-caption' => 'Confirmation',
 	'lightbox-share-email-ok-content' => '{{PLURAL:$1|Email has|Emails have}} been sent.',
```

Comment:
It is something different than useLang. userLang represents wgUserLanguage variable.